tests/integration/test_definitions/s3/integration_tests.py

```diff
@@ -82,7 +82,14 @@
         user_flow_script="tests/integration/docusaurus/deployment_patterns/aws_cloud_storage_pandas.py",
         data_context_dir=None,
         backend_dependencies=[BackendDependencies.AWS],
-    )
+    ),
+    # TODO: This will currently work locally, but the Azure CI/CD will need to be updated to enable
```

Comment:
This script currently works locally, and enabling Azure CI/CD to run the test is an upcoming task.